lib/screens/groups/groups.dart

```diff
@@ -10,6 +14,15 @@ class GroupsPage extends StatefulWidget {
 }
 
 class _GroupState extends State<GroupsPage> {
+  GroupsBloc _groupsBloc;
+
+  @override
+  void initState() {
```

Comment:
Override the dispose method and close the bloc in it

Reply:
update